sw/device/lib/dif/dif_clkmgr_unittest.cc

```diff
@@ -22,6 +22,54 @@ class ClkMgrTest : public Test, public MmioTest {
   dif_clkmgr_t clkmgr_ = {.base_addr = dev().region()};
 };
 
+class JitterEnableTest : public ClkMgrTest {};
```

Comment:
just this part will need to change due to #9001
but i can work on that separately.